packages/iris-grid/src/sidebar/TableSaver.tsx

```diff
@@ -165,7 +166,11 @@ export default class TableSaver extends PureComponent<
   cancelableSnapshots: (CancelablePromise<unknown> | null)[];
 
   // WritableStream is not supported in Firefox (also IE) yet. use ponyfillWritableStream instead
+
+  // TODO: Fix type error
+  // eslint-disable-next-line @typescript-eslint/strict-boolean-expressions
   WritableStream = window.WritableStream || ponyfillWritableStream;
+  //  WritableStream: typeof window.WritableStream | typeof ponyfillWritableStream;
```

Comment:
TODO?
And also use null coalescing operator.

Reply:
This is suppose to be the type declaration of `WritableStream`, which was broken from the TypeScript conversion.